src/desdemona/ui/table.cljs

```diff
@@ -14,7 +14,19 @@
   ((if (contains? coll k)
      disj conj) coll k))
 
-(defn columns-toggler-component
+(defn make-toggler-affix
+  []
+  (let [toggler (sel1 :#columns-toggler)
```

Comment:
Well, that poses an interesting question: I think there should be more than 1 such table, but then toggler affixing becomes trickier: either you affix togglers to all of them at once (which doesn't sound right) or you still have to look at 1 table at a time. I think it should be resuable that way now, but I don't think it's a hard requirement, so if it's a ton of work it's OK to skip it.